autoload/db_ui/schemas.vim

```diff
@@ -106,7 +110,7 @@ let s:oracle_args = join(
       \    ";\n"
       \ ).';'
 let s:oracle_foreign_key_query = "
-      \SELECT DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
+      \SELECT /*csv*/ DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
```

Comment:
Can you briefly explain why is this needed? It looks like a comment, but I'm assuming it has something to do with parsing.

Reply:
the /*csv*/ reads as a comment in sqlplus so it does nothing when executed by sqlplus.  In sqlcl (binary 'sql') it reads as a formatting hint so the output results are generated as comma delimited with " enclosures to make parsing easier.  I had to use the strip_quotes() function to strip the " enclosures generated by that format since there isn't a setting to remove the " enclosures.

Reply:
Should same thing be set in `s:oracle_schemes_tables_query` or that somehow works like this?

Reply:
It is set in s:oracle_schemes_tables_query and in the schemes query

Reply:
I'm referring to this variable https://github.com/kristijanhusak/vim-dadbod-ui/pull/173/files#diff-64a7375523d30316a3830832410ac86411a390c17e61c5765399c08fd1614b34R121 . I don't see `csv` comment added to it, so I'm wondering if `schemes_tables_query` will work.

Reply:
That was weird.  It was showing in my local but didn't push upstream.  I fixed and pushed a new commit.

Reply:
I see what happened.  I was working in a separate test branch and when I squashed commits I missed that change.  I checked the diffs and everything is good now.